### DISCUSSION TITLE
How to Optimize Font Size and Content Width to Minimize Blank Space on the Right Side

### DISCUSSION THREAD

**Discussion by zhi-xuan-chen** [academicpages/academicpages.github.io#2605] [Q&A]
**Describe the bug**
The right side of the website has a large blank area. How can I adjust the font size and content display width to reduce the blank space?

**Screenshots**
![image](https://github.com/user-attachments/assets/9de95c46-f0a2-4721-b610-008a06a95c62)

**Comment by jakesong15** [comment]
I've also had issues with some pages showing the blank space and some pages not showing the blank space, and managed to find a work-around solution for this (YMMV):

For pages where you want to **remove** the empty space on the right, begin the page files in your _pages directory with **layout: archive**
For pages where you want to **insert** the empty space on the right, begin the page files in your _pages directory with **layout: single**
